integration/container/resize_test.go

```diff
@@ -21,9 +19,6 @@ func TestResize(t *testing.T) {
 	apiClient := testEnv.APIClient()
 
 	cID := container.Run(ctx, t, apiClient, container.WithTty(true))
-
-	poll.WaitOn(t, container.IsInState(ctx, apiClient, cID, "running"), poll.WithDelay(100*time.Millisecond))
```

Comment:
Note to self: this is fine to remove because `containerStart` sets the `running` state before it exits (which is a part of the start API call) https://github.com/moby/moby/blob/152036f0aab931439c4e18647335956b071edd01/daemon/start.go#L215

Reply:
Ah, yes, I went down the same path to verify.

I _guess_ there may still be cases where a container exited early or other things going wrong, but I don't think the `poll.WaitOn` will help for that.

We also have `docker wait`, which may be a solution for other cases (instead of polling) 🤔 

In either case, I thought "let's remove this", and perhaps we should make a round across tests to see if there's other ones that do this (because these kind of checks can easily "spread" if people use existing tests as example)